content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,546 @@
+The relative path to the downloaded software is assumed to be the `sources` directory.
+{{< /note >}}
+
+### Create the sources directory
+
+```shell
+mkdir sources
+cd sources/
+```
+
+### Swap configuration
+
+The default behavior of kubelet is to fail to start if swap memory is detected on a node.
+This means that swap should either be disabled or tolerated by kubelet.
+
+{{< note >}}
+If you configure the kubelet to tolerate swap, the kubelet still configures Pods (and the
+containers in those Pods) not to use swap space. To find out how Pods can actually
+use the available swap, you can read more about
+[swap memory management](/docs/concepts/architecture/nodes/#swap-memory) on Linux nodes.
+{{< /note >}}
+
+If you have swap memory enabled, either disable it or add `failSwapOn: false` to the
+kubelet configuration file.
+
+To disable swap temporarily:
+
+```shell
+sudo swapoff -a
+```
+
+To make this change persistent across reboots:
+
+Make sure swap is disabled in `/etc/fstab` or `systemd.swap`, depending how it was configured on your system.
+
+### Enable IPv4 packet forwarding
+
+```shell
+sudo tee /etc/sysctl.d/k8s.conf <<EOF
+net.ipv4.ip_forward = 1
+EOF
+
+sudo sysctl --system
+```
+The output is similar to:
+
+```
+...
+* Applying /etc/sysctl.d/k8s.conf ...
+net.ipv4.ip_forward = 1
+* Applying /etc/sysctl.conf ...
+```
+
+### Network Plugins
+
+This tutorial suggests downloading CNI network plugins.
+
+Visit [releases](https://github.com/containernetworking/plugins/releases) (external link).
+
+{{< tabs name="download_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+Install:
+
+```shell
+sudo mkdir -p /opt/cni/bin
+```
+
+{{< tabs name="install_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+sudo tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+sudo tar Cxvf /opt/cni/bin/ cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+Quick test:
+
+```shell
+/opt/cni/bin/host-local --version
+```
+
+The output is similar to:
+
+```
+CNI host-local plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+Configure:
+
+```shell
+sudo mkdir -p /etc/cni
+
+sudo tee /etc/cni/cni.template <<EOF
+{
+  "name": "containerd",
+  "cniVersion": "1.0.0",
+  "plugins": [{
+    "type": "ptp",
+    "ipMasq": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "{{.PodCIDR}}",
+      "routes": [
+        {"dst": "0.0.0.0/0"}
+      ]
+    }
+  },{
+    "type": "portmap",
+    "capabilities": {
+      "portMappings": true
+    }
+  }]
+}
+EOF
+```
+
+### Container runtime
```

Comment:
It seems a little weird for us to reproduce all this containerd setup documentation here. Maybe it would be better to just link to the containerd setup guide, https://github.com/containerd/containerd/blob/main/docs/getting-started.md? You could also include a cri-o option: https://github.com/cri-o/cri-o/blob/main/install.md

Reply:
We've already reproduced the [container runtimes documentation](https://kubernetes.io/docs/setup/production-environment/container-runtimes/) in our own website. But maybe I could just link to that page and remove those steps. 

The idea of the tutorial was to have simple, step-by-step instructions all in one place for quick testing, though.

You're right. I'll add a section for CRI-O. We should strive to be agnostic, as much as possible.

Reply:
@tallclair, I was testing `cri-o` as the **container runtime**, thanks for the suggestion. It's interesting... If we use `cri-o` instead of `containerd`, we can skip both the **Network Plugins** installation & configuration and the **runc** installation sections.

`cri-o` has an [installer](https://github.com/cri-o/cri-o/blob/main/install.md) that includes the `cni-plugins`with a default configuration to make them work out-of-the-box, and the built-in `crun` to run containers, so no need for `runc` (it can use both, actually).

We could get rid of those sections and just leave the links in case somebody still wants to use `containerd` and omit the instructions and configuration files. Or just leave them, there's always somebody that's going to be unhappy because of our choice of runtimes.

If we do write instructions for both `containerd` _and_ `cri-o`, or vice versa, it's going to be a lengthier document, for sure. What do you think, @sftim?

Reply:
Done